tests/shared.py

```diff
@@ -336,12 +336,12 @@ def no_internet_connection():
         conn = socket.create_connection((host, 80), 2)
         conn.close()
         return False
-    except socket.gaierror:
+    except (socket.gaierror, TimeoutError):
```

Comment:
Sometimes I get timeouts from the `no_internet_connection()` function. This makes that also count as "not connection", but since I was actively connected to the internet, I don't think this function is working as expected.